src/xSTIR/cSTIR/include/sirf/STIR/stir_data_containers.h

```diff
@@ -160,6 +160,8 @@ namespace sirf {
 			return false;
 		}
 
+		virtual std::unique_ptr<PETAcquisitionData> get_subset(const std::vector<int>& views) const = 0;
```

Comment:
not sure why a pure virtual here. Is it not going to be the same for both?

Reply:
ok